TERMS.md

```diff
@@ -1,4 +1,114 @@
 
-in progress
+Thank you for choosing our service which allows people to transfer data ("Service"). The Service is provided by Least Authority TFA GmbH, Thaerstraße 28a, 10249 Berlin ("Least Authority", "we" or "us"). By using the Service, you conclude a binding agreement with us under the following terms and conditions. The Terms shall apply in the version current at the time of the first use of the Service as a framework agreement (§ 305 (3) of the German Civil Code (BGB)) and also to subsequent uses of the Service by the same person without having to refer to them again:
 
+**1. Scope of the Service**
+
+1.1 The Service allows a person ("Sender") to transfer data through the Service to another person ("Receiver"). For this purpose, we provide the Sender with a one-time code or other means to enable another person’s access to the content. The Sender may forward these means to the Receiver using any method the Sender  deems appropriate for this purpose. We do not forward the means to the Receiver as part of our Service. The Receiver may use the means to establish a one-time transfer of the content. 
+
+1.2 Both Sender and/or Receiver can use the Service without prior registration. 
+
+1.3 Both sending and receiving data through the Service is a use of the Service. You shall only use the Service in accordance with these Terms.
+
+1.4 We provide the Service "as is" and free of charge. These Terms do not grant you any right or claim with respect to any form, functionality and/or availability of the Service. 
+
+1.5 We especially reserve the right to alter, amend, suspend, or discontinue the Service at any time at our own discretion.
+
+1.6 It is technically possible to connect and use the backend servers used by  Destiny with applications other than Destiny. By using these servers with other applications, you agree to be bound to these same Terms.
+
+1.7 We offer our services in support of international human rights, including the rights to freedom of expression and privacy.  
+
+**2. Own responsibility**
+
+2.1 You are fully and solely responsible for the content that you transfer via our Service and the consequences of such transfer. 
+
+2.2 If the content that you transfer through the Service is subject to potential rights of third parties, you hereby undertake to ensure that you are authorized to transfer and use the content and that you do not infringe any third party rights.
+
+2.3 If you are a Sender, you must ensure that the transfer of content is in compliance with applicable law. As a Sender you especially ensure that the transfer complies with all applicable export control regulations. 
+
+**3. Prohibition of misuse**
+
+3.1 Misusing the Service or helping others to do so is strictly prohibited. You therefore agree that you will not use the Service in any harmful or malicious way. In particular, but not exclusively, you shall not do one of the following: 
+
+    3.1.1 transferring content that violates applicable law;
```

Comment:
Do we want to have tab spacing here, as this triggers to look like separate block?